dawn/src/dawn/Optimizer/OptimizerContext.cpp

```diff
@@ -350,7 +347,7 @@ class StencilDescStatementMapper : public iir::ASTVisitor {
     // This is the first time we encounter this variable. We have to make sure the name is not
     // already used in another scope!
 
-    int AccessID = metadata_.addStmt(context_.getOptions().KeepVarnames, stmt);
+    int AccessID = metadata_.addStmt(false, stmt);
```

Comment:
Should this use `StatementMapper.keepVarnames` or is that inaccessible here?

Reply:
Good idea. It was removed because Options were not accessible at that point, but as of #990 that is again the case.